packages/angular/cli/src/utilities/environment-options.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+function isPresent(variable: string | undefined): variable is string {
+  return typeof variable === 'string' && variable !== '';
+}
+
+function isDisabled(variable: string | undefined): boolean {
+  return isPresent(variable) && (variable === '0' || variable.toLowerCase() === 'false');
+}
+
+function isEnabled(variable: string | undefined): boolean {
+  return isPresent(variable) && (variable === '1' || variable.toLowerCase() === 'true');
+}
+
+const analyticsVariable = process.env['NG_CLI_ANALYTICS'];
+export const analyticsDisabled = isDisabled(analyticsVariable);
+
+const analyticsShareVariable = process.env['NG_CLI_ANALYTICS_SHARE'];
+export const analyticsShareDisabled = isDisabled(analyticsShareVariable);
+
+const ciVariable = process.env['CI'];
+export const isCI = isEnabled(ciVariable);
+
+const disableVersionCheckVariable = process.env['NG_DISABLE_VERSION_CHECK'];
+export const disableVersionCheck = isEnabled(disableVersionCheckVariable);
+
+const ngDebugEnvVariable = process.env['NG_DEBUG'];
+export const ngDebug = isEnabled(ngDebugEnvVariable);
```

Comment:
I'm fine with this approach, I agree it's a lot cleaner.

One minor nit: Consider inlining the `*Variable` values, now that they are only referenced once each.

Reply:
Yeah, let me do that